Vagrantfile

```diff
@@ -0,0 +1,54 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+  
+config.vm.box = "hashicorp/bionic64"
+config.vm.network "forwarded_port", guest: 5000, host: 5000
+config.vm.provision "shell", privileged: false, inline: <<-SHELL
+    sudo apt-get update
+    sudo apt-get update; sudo apt-get install -y build-essential libssl-dev zlib1g-dev \
+    libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm \
+    libncursesw5-dev xz-utils tk-dev libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev
+# TODO: Install pyenv prerequisites
+# TODO: Install pyenv
+    echo " installing pyenv."
+      if [ ! -d \"$HOME/.pyenv\" ]; then
+        git clone https://github.com/pyenv/pyenv.git $HOME/.pyenv                  
+      fi
+      echo "export pyenv root"  
+      # export PYENV_ROOT=\"$HOME//.pyenv/\"
+      # export PATH=\"$PYENV_ROOT/bin:$PATH\"
+        echo "Echoing step for error"
+      if [ -d \"$HOME/.pyenv\" ]; then
+          echo "pyenv already installed on filesystem."
+      fi
+      echo 'export PYENV_ROOT="$HOME/.pyenv"' >> ~/.profile
+      echo 'export PATH="$PYENV_ROOT/bin:$PATH"' >> ~/.profile
+      echo 'eval "$(pyenv init -)"' >> ~/.profile
+      source ~/.profile
+         
+           echo "End of pyenv"  
```

Comment:
Your indenting switches here - it doesn't break anything but makes this a shade harder to read